backend/src/main/java/com/tourplanner/backend/controller/impl/TourController.java

```diff
@@ -0,0 +1,51 @@
+package com.tourplanner.backend.controller.impl;
+
+import com.tourplanner.backend.controller.IGenericController;
+import com.tourplanner.backend.service.dto.TourDTO;
+import com.tourplanner.backend.service.IGenericService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/tour")
```

Comment:
The controllers are great, but I guess we need to stick to REST guidelines when it comes to naing endpoints. 
![image](https://github.com/imoalim/TourPlanner/assets/127953460/8cdc3e4b-3a2b-434c-91e6-dddfdede0899)